src/regs/gich/gich_eisr.rs

```diff
@@ -0,0 +1,97 @@
+//! End Interrupt Status Register, GICH_EISR
+//! The GICH_EISR characteristics are:
+//!
+//! ## Purpose
+//!
+//! Indicates which List registers have outstanding EOI maintenance interrupts.
+//!
+//! ## Configuration
+//!
+//! This register is present only when FEAT_GICv3_LEGACY is implemented and EL2 is implemented. Otherwise, direct accesses to GICH_EISR are RES0.
+//!
+//! This register is available when the GIC implementation supports interrupt virtualization.
+//!
+//! ## Attributes
+//!
+//! GICH_EISR is a 32-bit register.
+
+use tock_registers::register_bitfields;
+use tock_registers::registers::ReadOnly;
+
+register_bitfields! {u32,
+    pub GICH_EISR [
+        /// [31:16] Reserved, RES0.
+        Reserved31_16 OFFSET(16) NUMBITS(16) [],
+        /// [15:0] Status<n>
+        /// EOI maintenance interrupt status for List register <n>.
+        // Status [
+            Status15 OFFSET(15) NUMBITS(1) [
```

Comment:
same question as `GICH_APR`, do we need to seperate each bits?

it depends on our operation on this fields. in my opinion this will make the function more complex.

Reply:
OK